reference/channels.v3.yml

```
@@ -39,8 +39,8 @@ info:
     | `vend`            | `pos`                     |
     | `clover`          | `pos`                     |
     | `talech`          | `pos`                     |
-    | `facebook`        | `marketplace`,`marketing` |
-    | `instagram`       | `marketplace`,`marketing` |
+    | `facebook by meta`| `marketplace`,`marketing` |
+    | `facebook by instagram`| `marketplace`,`marketing` |
```

```suggestion
    | `instagram by meta`| `marketplace`,`marketing` |
```
Looking at the ticket, seems as though this should be `instagram by meta`